### DISCUSSION TITLE
Github pages when deploying changes font used on page to a random font

### DISCUSSION THREAD

**Discussion by zvonimirkonjevic** [community/community#41920] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Github pages when deploying changes font used on page to a random font, did anyone else had the same issue and how to fix it? Thanks in advance.

Here is the repository that has this issues: https://github.com/zvonimirkonjevic/rock-paper-scissors

**Comment by jfullstackdev** [comment, marked as answer]
I cannot see your deployed website

![image](https://user-images.githubusercontent.com/47092464/208145768-a235a2c4-6e68-4c90-ad4e-bdaa38c4d03d.png)

your root does not contain anything too 

![image](https://user-images.githubusercontent.com/47092464/208145850-b8d7861b-5479-4192-80ea-435e10538ae2.png)

**Reply by zvonimirkonjevic** [reply]
Hey i just deployed it again so you should be able to see it now

**Reply by jfullstackdev** [reply]
this is on my end 

![image](https://user-images.githubusercontent.com/47092464/208225520-a85c3078-2cc1-4190-b63e-d7e73e607dd0.png)

but you may want to check at the Developer Tools, this might be the one that is not loading for your fonts

![image](https://user-images.githubusercontent.com/47092464/208225538-115b27e1-f86d-407d-9fea-e476676ffe40.png)

**Reply by zvonimirkonjevic** [reply]
Will do thanks bc it loads the font when I host with live server in visual studio code

**Reply by jfullstackdev** [reply]
is that a local package you installed ? that is a different one when you deploy it on GitHub Pages vs VS Code, if say that's an online service, then for sure, it will run

**Reply by zvonimirkonjevic** [reply]
yeah, so i checked and found out that github when hosting my website doesn't include fonts folder in sources
![Screenshot from 2022-12-17 10-25-23](https://user-images.githubusercontent.com/65192988/208235427-59eaffe2-4dfc-40a0-bbb7-7bf8d284372b.png)

**Reply by zvonimirkonjevic** [reply]
do you have any idea why that would be the case? thanks in advance

**Reply by jfullstackdev** [reply]
we need to know whether it's loaded

when you click this 

https://zvonimirkonjevic.github.io/rock-paper-scissors/fonts/GorditaBold.otf

which is the actual URL for that font style, it's there because the file will be downloaded

but here is the problem

![image](https://user-images.githubusercontent.com/47092464/208246381-12d51991-400a-4ae0-8122-c0eb4e1eed75.png)

that is a relative path, which is quite tricky, try the absolute URL that I provided above

another problem might be, it's not plaintext, which I'm not very sure of how GitHub Pages is dealing with that

![image](https://user-images.githubusercontent.com/47092464/208246442-1a7164cb-0d07-4e15-b542-88e5b08b777c.png)

because GitHub Pages only runs directly HTML, CSS and JavaScript

**Reply by zvonimirkonjevic** [reply]
I'll change my relative link with this one that you provided and see what happens

**Reply by zvonimirkonjevic** [reply]
Okay yeah that fixed the problem, thanks a lot. Have a great day!
